manticore/native/cli.py

```diff
@@ -0,0 +1,34 @@
+from .manticore import Manticore
+from manticore.core.plugin import InstructionCounter, Visited, Tracer, RecordSymbolicBranches
+
+
+def main(args, logger_):
+    env = {key: val for key, val in [env[0].split('=') for env in args.env]}
+
+    m = Manticore(args.argv[0], argv=args.argv[1:], env=env, entry_symbol=args.entrysymbol,
+                  workspace_url=args.workspace, policy=args.policy,
+                  concrete_start=args.data, stdin_size=args.stdin_size)
+
+    m.verbosity(args.v)
```

Comment:
disclaimer: this was previously set as `Manticore.verbosity(args.v)` I believe we should make it an instance method instead of static method (this isn't covered by this PR; shall it?).